WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabViewModel.swift

```diff
@@ -170,13 +170,13 @@ extension ReaderTabViewModel {
 extension ReaderTabViewModel {
 
     func makeChildContentViewController(at index: Int) -> ReaderContentViewController? {
-        guard index < tabItems.count else {
-            return nil
+        guard tabItems.indices.contains(index) else {
```

Comment:
I don't know if it's clearer to use the [`[safe:]` subscript method](https://github.com/wordpress-mobile/WordPress-iOS/blob/0aa1510fffbd18a714dd390052cda8fe2826545c/WordPress/Classes/Extensions/CollectionType%2BHelpers.swift#L15) here:

```suggestion
        guard let tabItem = tabItems[safe: index] else {
```